app/src/test/java/org/mozilla/fenix/IntentReceiverActivityTest.kt

```diff
@@ -26,6 +28,90 @@ import org.robolectric.annotation.Config
 @Config(application = TestApplication::class)
 class IntentReceiverActivityTest {
 
+    @Test
+    fun `process intent with flag launched from history`() {
+        runBlockingTest {
+            testContext.settings().openLinksInAPrivateTab = false
+
+            val intent = Intent()
+            intent.flags = FLAG_ACTIVITY_LAUNCHED_FROM_HISTORY
+
+            `when`(testContext.components.intentProcessors.intentProcessor.process(intent)).thenReturn(true)
```

Comment:
Mockk uses a different system than Mockito. Instead of `when` use `every`:

```kt
every { testContext.components.intentProcessors.intentProcessor.process(intent) } returns true
```

Reply:
So, there is problem with 'every'. When i switch from 'when' to 'every' i have an following error: Suspension functions can be called only within coroutine body. I know that in Mockk is something like coEvery, but it's deprecated :(